packages/@aws-cdk/aws-ecs/lib/log-drivers/splunk-log-driver.ts

```diff
@@ -25,9 +25,16 @@ export enum SplunkLogFormat {
 export interface SplunkLogDriverProps extends BaseLogDriverProps {
   /**
    * Splunk HTTP Event Collector token.
+   * @deprecated Use {@link SplunkLogDriverProps.secretToken} instead.
    */
   readonly token: SecretValue;
 
+  /**
+   * Splunk HTTP Event Collector token (Secret).
```

Comment:
Can you update this description and explain that this field will add the token into the [SecretOptions](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-taskdefinition-containerdefinitions-logconfiguration.html#cfn-ecs-taskdefinition-logconfiguration-secretoptions) property and it will not be stored in plain text anywhere?

And add that at least one of this property or `token` is required.